src/main/java/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package strategy;
+
+import model.FruitTransaction;
+
+public interface OperationHandler {
+    void operation(FruitTransaction fruitTransaction);

```

Comment:
```suggestion
    void handle(FruitTransaction fruitTransaction);
```
we should use verbs for method names